x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/transforms/TransformTaskParams.java

```diff
@@ -50,28 +54,55 @@ public class TransformTaskParams implements SimpleDiffable<TransformTaskParams>,
         PARSER.declareLong(ConstructingObjectParser.optionalConstructorArg(), FROM);
         PARSER.declareString(ConstructingObjectParser.optionalConstructorArg(), FREQUENCY);
```

Comment:
Since you are using statically imported `UNATTENDED`, could you do the same with `FROM` and `FREQUENCY`?
This will make the code more consistent.